src/main/java/frc/robot/subsystems/Grabber.java

```diff
@@ -124,9 +115,9 @@ private void movePivot() {
     m_pivot.set(m_pivotPID.calculate(getPivotAngle(), m_lastPivotPosition - m_aimAngle));
   }
 
-  private void lockPivot() {
-    m_pivotLocked = true;
-  }
+  // private void lockPivot() {
+  //   m_pivotLocked = true; this is seemingly not used so I'm going to comment it out.
```

Comment:
Please remove this and all the unlock pivot stuff as well. It's an unused feature now